sdk/template/azure-template/CMakeLists.txt

```diff
@@ -19,12 +19,13 @@ include(global_compile_options)
 
 az_vcpkg_integrate()
 
-if(NOT AZ_ALL_LIBRARIES)
-  find_package(azure-core-cpp CONFIG QUIET)
-  if(NOT azure-core-cpp_FOUND)
-    find_package(azure-core-cpp REQUIRED)
-  endif()
-endif()
+# Disable dependency on azure-core until azure-core has released to vcpkg
```

Comment:
I assume we will not need this for the other packages as we plan to release them all together?

Reply:
From what I understood, this is required just for boot strapping and testing (since azure-template is what we use for dry runs). We shouldn't need to do this going forward, even if we don't release everything together. That said, we do need to ensure that packages like azure-core that client service libraries depend on are available on vcpkg before they release or are released simultaneously.